src/components/Conversation.tsx

```diff
@@ -0,0 +1,100 @@
+import React from "react";
+// import { useHistory } from "react-router-dom";
+import "../css/ConversationStyle.css";
+import "../css/ThankYouPage.css";
+import $ from "jquery";
+//import firebase from "../firebase.js"
+
+export default function Conversation() {
+    // export default class Conversation extends React.Component {
+    /*
+    constructor(props : PropTypes) {
+        super(props);
+        this.appendUserConvo = this.appendUserConvo.bind(this);
+		this.thankUser = this.thankUser.bind(this);
+    }
+
+*/
+    /* Unused
+    let history = useHistory();
+    function nextPath(path: any) {
+        history.push(path);
+    }
+    */
```

Comment:
You can delete this if you want. I most likely wont use it. It was a mistake on my part to not remove it before the initial pull request.